src/Omnimail.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Omnimail;
+
+use Omnimail\Common\Factory;
+
+/**
+ * Omnimail class
+ * This class was inspired by / copied from the Omnipay library.
+ *
+ * Provides static access to the gateway factory methods.  This is the
+ * recommended route for creation and establishment of mail gateway
+ * objects via the standard Factory.
+ *
+ * Example:
+ *
+ * <code>
+ *   // Create a gateway for the Silverpop mail client
+ *   // (routes to Factory::create)
+ *   $mailer= Omnimail::create('Silverpop');
+ *
+ *   // Get the mailer parameters.
+ *   $parameters = $mailer->getParameters();
+ * </code>
+ *
+ * @method static array all()
+ * @method static array replace(array $mailers)
+ * @method static string register(string $className)
+ * @method static array find()
+ * @method static array getSupportedMailers()
+ * @method static EmailSenderInterface create(string $class, array $parameters = [])
+ */
+class Omnimail
+{
+    const AMAZON_SES = '\Omnimail\AmazonSES';
```

Comment:
Yeah, it does not change the way you implemented it, but adds the possibility to use a constant to create a known provider.